portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -29,6 +42,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
   
+  
```

Comment:
Try to delete any random blank line from a commit if it adds nothing. Can cause some minor issues with merge conflicts down the road and prevents the code formatting from being truly standardized